maven-artifact/src/test/java/org/apache/maven/artifact/versioning/ComparableVersionTest.java

```diff
@@ -147,10 +154,10 @@ void testVersionsEqual() {
         checkVersionsEqual("1.0.0x", "1-x");
 
         // aliases
-        checkVersionsEqual("1ga", "1");
-        checkVersionsEqual("1release", "1");
-        checkVersionsEqual("1final", "1");
-        checkVersionsEqual("1cr", "1rc");
+        checkVersionsEqualOrder("1ga", "1");
+        checkVersionsEqualOrder("1release", "1");
+        checkVersionsEqualOrder("1final", "1");
```

Comment:
What you seem to have done is remove existing tests and replace them with new ones. Instead, leave the existing tests in place and unchanged. Then. add new tests in new methods.

We need to be really sure that existing behavior doesn't change unless we want it to.